spyder/app/tests/test_mainwindow.py

```diff
@@ -1860,5 +1860,25 @@ def test_pylint_follows_file(qtbot, tmpdir, main_window):
         assert fname == pylint_plugin.get_filename()
 
 
+def test_report_comms_error(qtbot, main_window):
+    """Test if a comms error is correctly displayed."""
+    CONF.set('main', 'show_internal_errors', True)
+    shell = main_window.ipyconsole.get_current_shellwidget()
+    qtbot.waitUntil(lambda: not shell._executing)
```

Comment:
I think the problem on Windows is related to this line. Please use the same line we use in all other tests to wait for the console to be fully ready to execute code:

```python
qtbot.waitUntil(lambda: shell._prompt_html is not None, timeout=SHELL_TIMEOUT)
```